README.md

```diff
@@ -134,15 +123,16 @@ Please see [SECURITY.md](./SECURITY.MD).
 
 ### Module
 
-See the Section Modules/Security.
+See the code in [modules/security](./contracts/modules/security).
 
-The Access Control is managed inside the module `AuthorizationModule`.
+Access control is managed thanks to the module `AuthorizationModule`.
 
-The module `OnlyDelegateCallModule` is a special module to insure that some functions (e.g. delegatecall and selfdestruct) can only be triggered through proxies when the contract is deployed with a Proxy.
+The module `OnlyDelegateCallModule` is a special module to insure that some functions (e.g., such as `delegatecall()`) can only be triggered through proxies when the contract is deployed with a proxy.
```

Comment:
why remove "and selfdestruct" ? This module was initially created to protect the function kill

Reply:
As `selfdestruct` wont be supported anymore, not the best example to give imho.

Reply:
Yes right, but when we will remove the self_destruct from the function kill, we will remove this module too...
We do not use delegatecall inside our contract.